data/actionsmap/yunohost.yml

```diff
@@ -1289,6 +1289,36 @@ tools:
 
 
 #############################
+#         Migrations        #
+#############################
+migrations:
```

Comment:
That raises the question of : are those migrations meant to be run automatically, or manually ?

On one hand, we'd like them to be automatic. On the other hand, some stuff are tricky like, for instance, the old letsencrypt_ynh app -> new builtin certmanager system, for which migrating is not trivial.